src/designer/elsa-workflows-studio/src/components/screens/workflow-instance-viewer/elsa-workflow-instance-journal/elsa-workflow-instance-journal.tsx

```diff
@@ -242,7 +243,7 @@ export class ElsaWorkflowInstanceJournal {
               <div class="elsa-relative elsa-flex elsa-space-x-3">
                 <div>
                   <span
-                    class="elsa-h-8 elsa-w-8 elsa-rounded-full elsa-bg-green-500 elsa-flex elsa-items-center elsa-justify-center elsa-ring-8 elsa-ring-white elsa-mr-1"
+                    class={`elsa-h-8 elsa-w-8 elsa-rounded-full elsa-bg-${eventColor}-500 elsa-flex elsa-items-center elsa-justify-center elsa-ring-8 elsa-ring-white elsa-mr-1`}
```

Comment:
I am guilty of this as well, but we shouldn't dynamically construct TailwindCSS classes like this, because they will be filtered out by the CSS cleanup phase in the build action.

Instead, it's better to assign the complete class value to the `eventColor` variable. Example (pseudo code):

`const eventColor = faulted ? 'elsa-bg-red-500' : 'elsa-bg-green-500'`

This way, the complete TailwindCSS class strings will be recognized.

Reply:
Nice, changing it.